### PR TITLE
Feature/select wiki source

### PR DESCRIPTION
**Pull request by jjm3x3**
This should handle #42 context in there!

### CODE REVIEW

File: app/contestants/page.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { getTeamList, ITeam } from "../utils/wikiQuery"
-import { wikiUrl, getWikipediaContestantData } from "../utils/wikiFetch"
+import { wikiUrl, apiUrl, getWikipediaContestantData } from "../utils/wikiFetch"
 
 export default async function Contestants() {
 
-    const wikiContestants = await getWikipediaContestantData()
+    const wikiContestants = await getWikipediaContestantData(apiUrl)
     const final = getTeamList(wikiContestants)
```
Comment (internette) [inline app/contestants/page.tsx:7]: You could convert these two lines into a util function called something like "getPageData"

File: app/scoring/page.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ function generateContestantRoundScores(contestantTeamsList: Team[], numberOfRoun
 
 export default async function Scoring() {
 
-    const wikiContestants = await getWikipediaContestantData()
+    const wikiContestants = await getWikipediaContestantData(apiUrl)
     const pageData = getTeamList(wikiContestants)
```
Comment (internette) [inline app/scoring/page.tsx:33]: And reuse the util here

**Review by internette — COMMENTED** [review]
Needs unit test

File: app/contestants/page.tsx (hunk 1 above)
Comment (jjm3x3) [reply app/contestants/page.tsx:7]: This is a decent idea, but I am not yet convinced that now is the right time to do that. [see Rule Of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))

File: app/scoring/page.tsx (hunk 2 above)
Comment (jjm3x3) [reply app/scoring/page.tsx:33]: Same as above.

**Comment by jjm3x3** [comment]
> Needs unit test

What should I unit test here?

**Comment by internette** [comment]
👍L(almost)GTM👍

**Pull request merged**